packages/sigint-collector/src/exercise/ExerciseManager.js

```
@@ -0,0 +1,377 @@
+            }
+        }
+    }
+    /**
+     * Trigger a scenario inject
+     */
+    triggerInject(exercise, inject) {
+        this.addEvent(exercise, 'INJECT', inject.description, inject.data);
+        // Generate appropriate data based on inject type
+        switch (inject.type) {
+            case 'SIGNAL':
+                const signal = this.signalGenerator.generateRFSignal(inject.data);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)